src/js/lib/admin-page/components/WipeMetadataCache.js

```diff
@@ -0,0 +1,20 @@
+import { __, sprintf } from '@wordpress/i18n';
+
+const WipeMetadataCache = ( { setting, label, note, onClick } ) => {
+	return (
+		<div className="container">
+			<div className="wipe-metadata-container">
+				<label className="wipe-metadata-label">{ __( label, 'wp-parsely' ) }</label>
+				<div className="wipe-metadata-controls">
+					<button type="button" className="wipe-metadata-button" onClick={ () => onClick( true ) }>{ __( 'Wipe your metadata?', 'wp-parsely' ) }</button>
+					{ __( note, 'wp-parsely' ) }
+					<div className="wipe-metadata-warning">
+						{ setting.parsely_wipe_metadata_cache ? <h1>{ __( 'DANGER: METADATA WILL BE WIPED UPON FORM SUBMISSION', 'wp-parsely' ) }</h1> : '' }
```

Comment:
Why is `h1` being used? It's not the title of the admin page. If it's an important paragraph, then visually style it as so, without using incorrect semantics.